test_entities.py

```diff
@@ -21,8 +21,8 @@ def test_name() -> None:
 
 
 def test_description() -> None:
-    description = str(Description("My Description\\nWith two lines"))
-    assert description == '(description "My Description\\nWith two lines")'
+    description = str(Description("My Description\\nWith two \" lines"))
+    assert description == '(description "My Description\\nWith two \\\" lines")'
```

Comment:
Why does it need three `\` here? There shouldn't be a need to escape the `"`.
```suggestion
    assert description == '(description "My Description\\nWith two \\" lines")'
```

Reply:
True.